content/zh-cn/docs/tutorials/services/source-ip.md

```diff
@@ -1,122 +1,206 @@
+title: Using Source IP
+content_type: tutorial
+min-kubernetes-server-version: v1.5
+-->
 
 <!-- overview -->
 
+<!-- 
+Applications running in a Kubernetes cluster find and communicate with each
+other, and the outside world, through the Service abstraction. This document
+explains what happens to the source IP of packets sent to different types
+of Services, and how you can toggle this behavior according to your needs.
+-->
+运行在 Kubernetes 集群中的应用程序通过 Service 抽象发现彼此并相互通信，它们也用 Service 与外部世界通信。
+本文解释了发送到不同类型 Service 的数据包的源 IP 会发生什么情况，以及如何根据需要切换此行为。
 
-Kubernetes 集群中运行的应用通过 Service 抽象来互相查找、通信和与外部世界沟通。本文介绍被发送到不同类型 Services 的数据包源 IP 的变化过程，你可以根据你的需求改变这些行为。
+## {{% heading "prerequisites" %}}
 
+<!-- 
+### Terminology
 
+This document makes use of the following terms:
+-->
+## 术语表  {#terminology}
 
-## {{% heading "prerequisites" %}}
+本文使用了下列术语：
 
+{{< comment >}}
+<!--
+If localizing this section, link to the equivalent Wikipedia pages for
+the target localization.
+-->
+如果本地化此部分，请链接到目标本地化的等效 Wikipedia 页面。
+{{< /comment >}}
 
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+<!--
+[NAT](https://en.wikipedia.org/wiki/Network_address_translation)
+: network address translation
 
+[Source NAT](https://en.wikipedia.org/wiki/Network_address_translation#SNAT)
+: replacing the source IP on a packet; in this page, that usually means replacing with the IP address of a node.
 
-## 术语表
+[Destination NAT](https://en.wikipedia.org/wiki/Network_address_translation#DNAT)
+: replacing the destination IP on a packet; in this page, that usually means replacing with the IP address of a {{< glossary_tooltip term_id="pod" >}}
 
+[VIP](/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies)
+: a virtual IP address, such as the one assigned to every {{< glossary_tooltip text="Service" term_id="service" >}} in Kubernetes
 
-本文使用了下列术语：
+[kube-proxy](/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies)
+: a network daemon that orchestrates Service VIP management on every node
+-->
+[NAT](https://zh.wikipedia.org/wiki/%E7%BD%91%E7%BB%9C%E5%9C%B0%E5%9D%80%E8%BD%AC%E6%8D%A2)
+: 网络地址转换
 
-* [NAT](https://en.wikipedia.org/wiki/Network_address_translation): 网络地址转换
-* [Source NAT](https://en.wikipedia.org/wiki/Network_address_translation#SNAT): 替换数据包的源 IP, 通常为节点的 IP
-* [Destination NAT](https://en.wikipedia.org/wiki/Network_address_translation#DNAT): 替换数据包的目的 IP, 通常为 Pod 的 IP
-* [VIP](/zh/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies): 一个虚拟 IP, 例如分配给每个 Kubernetes Service 的 IP
-* [Kube-proxy](/zh/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies): 一个网络守护程序，在每个节点上协调 Service VIP 管理
+[Source NAT](https://en.wikipedia.org/wiki/Network_address_translation#SNAT)
+: 替换数据包上的源 IP；在本页面中，这通常意味着替换为节点的 IP 地址
 
+[Destination NAT](https://en.wikipedia.org/wiki/Network_address_translation#DNAT)
+：替换数据包上的目标 IP；在本页面中，这通常意味着替换为 {{<glossary_tooltip text="Pod" term_id="pod" >}} 的 IP 地址
```

Comment:
The format is broken, please refer to [preview page](https://deploy-preview-34237--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/services/source-ip/#terminology).
```suggestion
: 替换数据包上的目标 IP；在本页面中，这通常意味着替换为 {{<glossary_tooltip text="Pod" term_id="pod" >}} 的 IP 地址
```

Reply:
thanks.